modules/git/submodule.go

```diff
@@ -76,7 +76,7 @@ func getRefURL(refURL, urlPrefix, repoFullName string) string {
 				pth = "/" + pth
 			}
 
-			if urlPrefixHostname == refHostname {
+			if urlPrefixHostname == refHostname || refHostname == sshDomain {
```

Comment:
Heya, do you think we should check the schemes here?

Reply:
This section is only for SCP URL that doesn't have scheme -- below starting on line 96 we do some scheme check